core/kazoo_services/src/kz_services.erl

```diff
@@ -1168,8 +1168,14 @@ get_service_modules() ->
 
 -spec get_filesystem_service_modules() -> atoms().
 get_filesystem_service_modules() ->
+    get_filesystem_service_modules(filelib:wildcard([code:lib_dir('kazoo_services'), "/src/services/*.erl"])).
```

Comment:
@lazedo We are going to need a new approach here... When the release is packaged it does not contain the src/ directory.